package.json

```diff
@@ -9,7 +9,9 @@
   },
   "author": "",
   "license": "ISC",
-  "dependencies": {},
+  "dependencies": {
```

Comment:
Looks like you ran `npm install i`. Remember is it `npm install` or `npm i` for short. Not both.